README.md

```diff
@@ -1 +1,5 @@
-# projecttrialrev
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please make your readme file more descriptive. You should try adding more sections to the readme like what your skills are , your contacts and social media .

Reply:
Brilliant job @PrinceBadru. But let's add a reason why we are asking for this change. For example, to make our project more professional or instructive. 👨‍💼 [OPTIONAL] And to help students, we can add a link to the [Readme template](https://github.com/microverseinc/readme-template)